From ba8507d3a2f25fd43dd242cacbddb8c86a039a5f Mon Sep 17 00:00:00 2001
From: "Lu, Shenghua" <shenghua.lu@intel.com>
Date: Mon, 17 Dec 2012 14:54:29 +0800
Subject: [PATCH 13/18] [PORT FROM MAIN] [UX] Add a interface function for
 cursor plane showing

BZ:52060

Add setFrameCount() function for cursor A enable/disable,
This function is controlled by application level.
Setting->Developer options->Show frame count
By default, it is desabled.

Change-Id: If97be87dc0e81c77e58e38928b405e4f6347e477
Signed-off-by: Shenghua Lu <shenghua.lu@intel.com>
Reviewed-on: http://android.intel.com:8080/81965
Reviewed-by: Yao, Yong <yong.yao@intel.com>
Reviewed-by: Li, XiaojingX <xiaojingx.li@intel.com>
Tested-by: Li, XiaojingX <xiaojingx.li@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Tested-by: cactus <cactus@intel.com>
---
 include/hardware/fb.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/include/hardware/fb.h b/include/hardware/fb.h
index 135e4aa..33beab9 100644
--- a/include/hardware/fb.h
+++ b/include/hardware/fb.h
@@ -143,7 +143,7 @@ typedef struct framebuffer_device_t {
      * Returns 0 on success or -errno on error.
      */
     int (*enableScreen)(struct framebuffer_device_t* dev, int enable);
-
+    int (*setFramecount)(int cmd, int count, int x, int y);
     void* reserved_proc[6];
 
 } framebuffer_device_t;
-- 
1.7.9

